Microsoft.Toolkit.Uwp.SampleApp/SamplePages/TextToolbar/TextToolbarPage.xaml.cs

```diff
@@ -157,5 +188,24 @@ private void EditZone_TextChanged(object sender, Windows.UI.Xaml.RoutedEventArgs
                 _previewer.Text = string.IsNullOrWhiteSpace(text) ? "Nothing to Preview" : text;
             }
         }
+
+        private int DemoCounter { get; set; } = 0;
+
+        private ValueHolder ToolbarFormat
+        {
+            get
+            {
+                var sample = DataContext as Sample;
```

Comment:
You need to check for null here unless you're sure then (cast) don't (as).
Same thing for the if(properties.TryGetValue), properties could be null because of the as.